index.html

```diff
@@ -1,51 +1,37 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Deanna Nguyen | Product Designer</title>
-    <link rel="shortcut icon" type="image/x-icon" href="images/favicon.png"/>
-
-    <!-- BOOTSTRAP -->
-    <link href="css/bootstrap.min.css" rel="stylesheet">
-    <link rel="stylesheet" href="font-awesome-4.4.0/css/font-awesome.min.css">
-    <link rel="stylesheet" type="text/css" href="css/styles.css">
-    <link rel="stylesheet" type="text/css" href="css/desktop.css">
-    <link rel="stylesheet" type="text/css" href="css/laptops.css">
-    <link rel="stylesheet" type="text/css" href="css/landscapephones.css">
-    <link rel="stylesheet" type="text/css" href="css/portraitphones.css">
-  </head>
-
-  <body>
-  	<nav class="navbar sticky-top navbar-expand-sm">
-		<div class="container">
-			<!-- BRAND -->
-			<a class="navbar-brand" href="http://deannanguyen.com">DEANNA</a>
-
-  			<!-- LINKS -->
-	  		<div class="collapse navbar-collapse justify-content-end">
-		  		<ul class="navbar-nav">
-		  			<li class="nav-item">
-		  				<a class="nav-link" href="#">LINK ONE</a>
-		  			</li>
-		  		</ul>
-
-		  		<ul class="navbar-nav">
-		  			<li class="nav-item">
-		  				<a class="nav-link" href="#">LINK TWO</a>
-		  			</li>
-		  		</ul>
-
-		  		<ul class="navbar-nav">
-		  			<li class="nav-item">
-		  				<a class="nav-link" href="#">LINK THREE</a>
-		  			</li>
-		  		</ul>
-		  	</div>
-		</div>
-  	</nav>
-  </body>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Deanna Nguyen | Product Designer</title>
+        <link rel="shortcut icon" type="image/x-icon" href="images/favicon.png"/>
 
+        <!-- BOOTSTRAP -->
+        <link href="css/bootstrap.min.css" rel="stylesheet">
+        <link rel="stylesheet" type="text/css" href="css/styles.css">
+    </head>
 
+    <body>
+        <div id="app">
+            <nav class="navbar navbar-dark bg-dark sticky-top navbar-expand-sm">
+                <div class="container">
+                    <!-- BRAND -->
+                    <a class="navbar-brand" href="#">DEANNA</a>
+                    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon"></span>
+                    </button>
+                    <!-- LINKS -->
+                    <div id="navbarNav" class="collapse navbar-collapse justify-content-end">
```

Comment:
Cleaned up the navbar-nav code block on line 26 so it is easier to work with. 

However if you did want to do it the way you originally chose with the list (<ul> / <li>) approach this would be the way:
`
    <ul class="navbar-nav">
      <li class="nav-item">
        <a class="nav-link" href="#">Link 1</a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#">Link 2</a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#">Link 3</a>
      </li>
    </ul>
`

The semantic HTML tag of <ul> stands for 'unordered list'.
The semantic HTML tag of <li> stands for 'list item'.

If you notice in the example above, one <ul> has many <li>. In your implementation, one <ul> has one <li>.

The tradeoff with my code compared to the code that you chose is that my code might be slightly easier to work with but your code has better accessibility for blind users.